python/python/glide/async_commands/standalone_commands.py

```diff
@@ -459,3 +459,44 @@ async def flushall(self, flush_mode: Optional[FlushMode] = None) -> TOK:
             TOK,
             await self._execute_command(RequestType.FlushAll, args),
         )
+
+    async def copy(
+        self,
+        source: str,
+        destination: str,
+        destinationDB: Optional[int] = None,
+        replace: Optional[bool] = None,
+    ) -> bool:
+        """
+        Copies the value stored at the `source` to the `destination` key on `destinationDB`. When `replace` is true,
+        removes the `destination` key first if it already exists, otherwise performs no action.
+
+        See https://valkey.io/commands/copy for more details.
+
+        Args:
+            source (str): The key to the source value.
+            destination (str): The key where the value should be copied to.
+            destinationDB (Optional[int]): The alternative logical database index for the destination key.
+            replace (Optional[bool]): If the destination key should be removed before copying the value to it.
+
+        Returns:
+            bool: True if the source was copied. Otherwise, return False.
+
+        Examples:
+            >>> await client.set("source", "sheep")
+            >>> await client.copy("source", "destination")
+                True # Source was copied
+            >>> await client.get("destination")
+                "sheep"
+
+        Since: Redis version 6.2.0.
+        """
+        args = [source, destination]
+        if type(destinationDB) is int:
```

Comment:
I'm not sure if its better to keep this line as is or change it to `if destinationDB is not None`. With this code, if the user passes in the wrong type for destinationType (eg a bool) that invalid argument will be ignored. With `if destinationDB is not None` the user will get an error. I know we changed to `if type(destinationDB) is int` to guard against the user passing an incorrect type but I'm not sure if its up to us to handle this scenario when the signature specifies the type that we expect